tern/analyze/docker/dockerfile.py

```diff
@@ -112,6 +112,26 @@ def expand_vars(dfobj):
             replace_env(dfobj.prev_env, obj)
 
 
+def expand_arg(dfobj):
+    '''Replace the ARG variables with their values if known
+    dfobj: the Dockerfile object created using get_dockerfile_obj'''
+    # get arg dictionary
+    arg_dict = {}
+    for instruction_desc in dfobj.structure:
+        instruction = instruction_desc['instruction']
+        if instruction == 'ARG':
```

Comment:
Instead of creating a new `instruction` variable, can you just check if `instruction_desc['instruction'] == 'ARG'` ?